file_formats/fip.md

```diff
@@ -20,39 +20,51 @@ For example:
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣📂 SoftwareEvents
+┃ ┣📜RepositoryStatus.json
+┃ ┗📜Region.json
+┣ green.csv
+┣ red.csv
+┣ iso.csv
+┣ green.bin
+┣ red.bin
+┣ iso.bin
+┣ roi_green_iso.csv
+┣ roi_red.csv
+┣ camera_green_iso_metadata.csv
+┗ camera_red_metadata.csv
 ```
-The file names are `FIP_[channel-name]_[datetime].csv`.  The `[channel-name]` token can be one of the following:
 
-* `DataG`: data for green channel
-* `DataR`: data for red channel
-* `DataIso`: data for isosbestic channel
+#### Photometry CSV files
 
-#### Photometry Readout CSV files
+Data files are named by the color of the channel.
+
+* `green.csv`: data for green channel
+* `red.csv`: data for red channel
+* `iso.csv`: data for isosbestic channel
 
 These files contain photometry readouts. Each row describes the average signal of pixels within each ROI for a single video frame. The values are computed online during data acquisition. The files have no headers. Each column is a timeseries, ordered as follows:
 
-* `timestamp` (software timestamp, in milliseconds, total time of the day)
+* `SoftwareTS` (software timestamp, in milliseconds, total time of the day)
 * `ROI0` (corresponding to fiber branch1) values
 * `ROI1` (corresponding to fiber branch2) values
 * `...`  (depending on how many fibers are used; in most of the experiments: ROI0-3/4branches)
-* `Blank ROI`: CMOS dark count floor signal
+* `Background`: CMOS dark count floor signal
+* `HarpTS` (Harp timestamp, in XXXunit, from Harp device reference)
+
 
 #### ROI Coordinate CSV files
 
 The ROI CSV files contain the vector representation of the ROIs used to integrate BIN video signal. The files have no headers. 
 There is one CSV file per camera, corresponding to G and Iso (time-multiplexing) and the other camera is recording only R. 
 The CSV files can be used to reconstitute images like this:
 
+* `roi_green_iso.csv`: ROI metadata for the green and isosbestic channels
+* `roi_red.csv`: ROI metadata for the red channel
+
 ![image](https://github.com/user-attachments/assets/30900798-5d51-43ba-99fc-41b07d4a75dd)
 
+
 Each row of a CSV is a point position in an ROI. The columns are, in this order:
```

Comment:
Add column headers here too.

Reply:
I think it is fine to not guarantee order.

Reply:
Can we add that to the document?

Reply:
Yep